c2corg_api/search/sync.py

```diff
@@ -62,13 +64,27 @@ def remove_doc():
             # https://github.com/c2corg/v6_api/issues/89
             DBSession.refresh(document)
 
+        is_user_profile = isinstance(document, UserProfile)
+        if is_user_profile:
+            # set user login + full-name as document title so that it can
+            # be searched
+            user_login, user_name = DBSession. \
+                query(User.name, User.username). \
+                filter(User.id == document.document_id). \
+                first()
+            user_title = '{0} {1}'.format(user_name or '', user_login or '')
```

Comment:
> We would have to store the usernames on the outing in ES.

I didn't mean that we needed to search outings by the associated users (maybe useful I don't know yet). I meant that we would probably have to use ES to pick the users to associate when editing an outing.